recipes/Prodigal/meta.yaml

```diff
@@ -0,0 +1,26 @@
+package:
+  name: prodigal
+  version: "2.6.2"
+
+source:
+  git_rev: v2.6.2
+  git_url: https://github.com/hyattpd/Prodigal.git
+
+build:
+  detect_binary_files_with_prefix: True
+ 
+requirements:
+  build:
+    - gcc
+  run:
+    - libgcc
```

Comment:
@Jorge, not sure. only tested on infernal and hmmer. I can remove it to see.

On Mon, Oct 19, 2015 at 1:53 PM, Jorge Cañardo Alastuey <
notifications@github.com> wrote:

> In recipes/Prodigal/meta.yaml
> <https://github.com/biocore/conda-recipes/pull/5#discussion_r42424453>:
>
> > +package:
> > +  name: prodigal
> > +  version: "2.6.2"
> > +
> > +source:
> > +  git_rev: v2.6.2
> > +  git_url: https://github.com/hyattpd/Prodigal.git
> > +
> > +build:
> > +  detect_binary_files_with_prefix: True
> > +
> > +requirements:
> > +  build:
> > +    - gcc
> > +  run:
> > +    - libgcc
>
> Is libgcc needed at all here? I added it to be safe, haven't tested
> without it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/conda-recipes/pull/5/files#r42424453>.
>